File: src/main/java/by/forward/forward_system/core/dto/messenger/v3/V3SearchOrderReviewDto.java
package by.forward.forward_system.core.dto.messenger.v3;

import by.forward.forward_system.core.enums.OrderStatus;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class V3SearchOrderReviewDto {
    private Long orderId;
    private String orderTechNumber;
    private OrderStatus status;
    private String statusRusName;
    private List<V3OrderReviewDto> reviews;
}

File: forward-system-frontend/src/core/dto/V3SearchOrderReviewDto.ts
import type {OrderStatusType} from "@/core/type/OrderStatusType.ts";
import type {OrderReviewDto} from "@/core/dto/OrderReviewDto.ts";

export interface V3SearchOrderReviewDto {
    orderId: number
    orderTechNumber: string
    status: OrderStatusType
    statusRusName: string
    reviews: OrderReviewDto[]
}